spec/models/manageiq/providers/openstack/cloud_manager_spec.rb

```diff
@@ -154,7 +154,7 @@ def lastlog
         :default_security_protocol => 'non-ssl',
         :default_userid            => 'admin',
         :default_hostname          => 'address',
-        :default_api_port          => '5000'
+        :default_port              => '5000'
```

Comment:
NOTE this test was passing for the same reason https://github.com/ManageIQ/manageiq-providers-openstack/pull/696 was failing, it was only ever checking what we consider as the default port, port 5000.

Added a spec that tests a different port: https://github.com/ManageIQ/manageiq-providers-openstack/pull/698/files#diff-459deb5359543a21bcb0aa2a4fed7033e624bb845a5fafe35e4aee1ea9165b9dR191